devfile.yaml

```diff
@@ -0,0 +1,72 @@
+commands:
```

Comment:
What is this file and what is it doing? 😅

Should there be additional readme documentation explaining what this is and how to use it? Is this an optional component/tool that is available for developers, but not necessary to run the project?

Reply:
Sorry, this should be in the odo branch. It's not doing anything. Anything odo-related was accidentally pushed into this, I'll remove that stuff, it's not relevant to this PR